tests/tkupchyn/homework_02/three_words.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+
+def three_in_row(text):
+    count = 0
+    for elem in text.split():
+        if not elem.isdigit():
+            count += 1
+        else:
+            count = 0
+        if count == 3:
+            return True
+    return False
```

Comment:
try to provide additional solution using iterating by indexes and using slices e.g.:

```

for i in range(len(words)):
    three_words = words[i: i +3]
```

Reply:
Added alternative solution, moved the function to separate file.